### ISSUE TITLE
google.com: breakage

### ISSUE THREAD

**Issue by neviln** [uBlockOrigin/uAssets#11476]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.google.com/search?q=gabapentin&rlz=1C1CHBD_enCA831CA831&biw=1912&bih=903&sxsrf=APq-WBtV28TqtHAwaRJeQHnS6IuK0yB2ew%3A1643242594209&ei=YuTxYdiVDLWU0PEP8JCkuAc&ved=0ahUKEwjYzu3Y09D1AhU1CjQIHXAICXcQ4dUDCA8&uact=5&oq=gabapentin&gs_lcp=Cgdnd3Mtd2l6EAMyBAgAEEMyCAgAELEDEJECMgQIABBDMgcIABCxAxBDMgQIABBDMgQIABBDMggIABCABBCxAzILCAAQgAQQsQMQgwEyCAgAEIAEELEDMggIABCABBCxAzoECAAQRzoECCMQJzoLCC4QgAQQxwEQowI6EQguEIAEELEDEIMBEMcBENEDOgUIABCRAjoFCC4QkQI6DgguEIAEELEDEMcBEKMCOgcILhCxAxBDOgsILhCxAxDHARDRAzoICC4QgAQQsQM6CwguEIAEEMcBEK8BOgUIABCABEoECEEYAEoECEYYAFD2NljcP2DrRWgAcAZ4AYABxwaIAesNkgEHNC41LjYtMZgBAKABAcgBCMABAQ&sclient=gws-wiz`

### Category

breakage

### Describe the issue

The "People Also Ask" section on Google search results page doesn't work. The section loads on the page, but the pulldowns don't do anything when you click the questions. 

Once I disabled uBlock, the pulldowns worked. 

This has always been work as long as I can remember, but I noticed the issue today.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.40.8
Chromium: 97
filterset (summary): 
  network: 79112
  cosmetic: 42088
  scriptlet: 19256
  html: 0
listset (total-discarded, last updated): 
  added: 
    ublock-annoyances: 4365-8, 56m
  default: 
    easylist: 63525-9, 56m
    easyprivacy: 26670-17, 56m
    plowe-0: 3679-817, 56m
    ublock-abuse: 76-0, 56m
    ublock-badware: 3816-69, 56m
    ublock-filters: 30282-156, 56m
    ublock-privacy: 195-2, 56m
    ublock-unbreak: 1699-39, 56m
    urlhaus-1: 7417-0, 56m
filterset (user): [empty]
trustedset: 
  added: [array of 6 redacted]
switchRuleset: 
  added: [array of 2 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  launchToReadiness: 762 ms (selfie)
popupPanel: 
  blocked: 10
```
</details>

**Comment by Procyon-b** [comment]
It works on my side. I can replicate your problem only if I disable _inline script_ or _1st-party script_

**Comment by neviln** [comment]
Would you happen to know what could have caused "inline script" or "1st-party script" to be disabled on my end? I hadn't touched any of uBlock's settings or filters in over a year... aside from the occasional time I'd click the power button to bypass uBlock for a specific site.

**Comment by uBlock-user** [comment]
Reset to default settings/lists + disable all other extensions.

**Comment by neviln** [comment]
UPDATE: It appears the issue is with Chrome and not uBlock. It must have been coincidental that when I disabled uBlock and refreshed the page earlier, the "People Also Ask" (PAA) worked. 
I've now tried multiple times to disable uBlock and the PAA pulldowns are still not working. After doing more digging, it appears I'm not the only one experiencing this issue.

Nonetheless, thank you for trying to help me out.
